service_account_file.json

```diff
@@ -0,0 +1,3 @@
+{
+}
```

Comment:
Should this be committed?

Reply:
I suggest we rename it to `google_service_account_file.json`. What do you think?